### PR TITLE
feat: support performance timer for more precise milliseconds

### PR DESCRIPTION
**Pull request by fengmk2**
e.g.: 1.375ms better than 1ms

##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

File: benchmark/index.js [hunk 1]
```diff
@@ -78,6 +99,81 @@ suite
   })
   .run({ async: false });
 
+// node version: v16.13.0, date: Sat Nov 20 2021 14:05:24 GMT+0800 (中国标准时间)
+// Starting...
+// 10 tests completed.
+
+// logger.error(err)                                           x   378,794 ops/sec ±1.35% (91 runs sampled)
```
Comment (fengmk2) [inline benchmark/index.js:106]: 只要升级 node 到 16，性能就翻倍。

**Comment by fengmk2** [comment]
@popomore @atian25 帮忙看看？

File: test/lib/egg/context_logger.test.js [hunk 2]
```diff
@@ -131,8 +135,23 @@ describe('test/lib/egg/context_logger.test.js', () => {
       .expect('done', err => {
         should.not.exists(err);
         const body = fs.readFileSync(filepath, 'utf8');
-        const m = body.match(/\/\d*ms/g);
-        (parseInt(m[1].substring(1)) > parseInt(m[0].substring(1))).should.equal(true);
+        const m = body.match(/\/\d+ms/g);
+        console.log(m);
```
Comment (popomore) [inline test/lib/egg/context_logger.test.js:139]: console

**Review by atian25 — APPROVED** [review]

File: .github/workflows/nodejs.yml [hunk 3]
```diff
@@ -18,8 +22,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        node-version: [8, 10, 12, 14]
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        node-version: [8, 10, 12, 14, 16]
+        os: [ubuntu-latest, macos-latest]
```
[the remark this answers is not in the document]
Comment (fengmk2) [reply .github/workflows/nodejs.yml:26]: 删文件我们单测下很难稳定跑过。先不测试了。

**Comment by fengmk2** [comment]
2.7.0

**Pull request merged**